### DISCUSSION TITLE
AdvancedMarker operates much slower than Maker

### DISCUSSION THREAD

**Discussion by Ferla04** [visgl/react-google-maps#291] [Q&A]
I am currently using Marker to render approximately 2100 data, when using AdvancedMarker it is slower, maybe it is not the most optimal to show all the data at the same time, maybe you don't have a functionality to group Markers?

**Comment by cuongdtt** [comment]
Hi @Ferla04 , I have the same issue with AdvancedMarker (it works fine with Marker) and this workaround https://github.com/visgl/react-google-maps/issues/171#issuecomment-1903611841 works for me
